packages/lib/src/vis/shared/TooltipMesh.tsx

```diff
@@ -38,15 +38,13 @@ function TooltipMesh(props: Props) {
   // When panning, events are handled and stopped by texture mesh and do not reach this mesh (which is behind)
   const onPointerMove = useCallback(
     (evt: ThreeEvent<PointerEvent>) => {
-      const { zoom } = camera;
-      const projectedPoint = camera.worldToLocal(evt.unprojectedPoint.clone());
-
       const abscissaCoord = abscissaScale.invert(evt.unprojectedPoint.x);
       const ordinateCoord = ordinateScale.invert(evt.unprojectedPoint.y);
 
+      const projectedPoint = evt.unprojectedPoint.clone().project(camera);
       showTooltip({
-        tooltipLeft: projectedPoint.x * zoom + width / 2,
-        tooltipTop: -projectedPoint.y * zoom + height / 2,
+        tooltipLeft: (width / 2) * (projectedPoint.x + 1),
+        tooltipTop: -(height / 2) * (projectedPoint.y - 1),
```

Comment:
As `projectedPoint` is now in the normalized space, I don't have to care about the camera parameter but only to move from the normalized space into the HTML space.

I shift the center (HTML `0,0` is the top) and scale by `width / 2` in X and `height / 2` in Y. There is a sign change for Y as the Y-axis needs to be inverted.